streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -502,6 +517,13 @@ public StreamsConfig(final Map<?, ?> props) {
         consumerProps.put(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, originals().get(BOOTSTRAP_SERVERS_CONFIG));
         // remove deprecate ZK config
         consumerProps.remove(ZOOKEEPER_CONNECT_CONFIG);
+        if (eosEnabled) {
+            if (clientProvidedProps.containsKey(ConsumerConfig.ISOLATION_LEVEL_CONFIG)) {
```

Comment:
Yes - my opinion in this case is to log it and move on. I don't see it as exceptional, but whatever. I'm happy either way